pkg/config/config.go

```diff
@@ -42,6 +42,9 @@ type Config struct {
 	// MaxArrayValues is the maximum number of array items that the commands
 	// print, locals, args and vars should read (in verbose mode).
 	MaxArrayValues *int `yaml:"max-array-values,omitempty"`
+	// MaxVariableRecurse is output evaluation depth of of nested struct members, array and
```

Comment:
nit: two "of"'s.

Reply:
Not understood, where double s ?

Reply:
No I mean, it says "of of" instead of just "of".